scss/_patterns_grid.scss

```diff
@@ -98,23 +75,23 @@
 
   // span full grid on other breakpoints; (to match block level element behaviour)
   @for $i from 1 through $grid-columns-small {
-    .#{$grid-small-col-prefix}#{$i} {
+    .row .#{$grid-small-col-prefix}#{$i} {
       @extend %span-full-grid-on-tablet;
       @extend %span-full-grid-on-desktop;
       @extend %display-block;
     }
   }
 
   @for $i from 1 through $grid-columns-medium {
-    .#{$grid-medium-col-prefix}#{$i} {
+    .row .#{$grid-medium-col-prefix}#{$i} {
       @extend %span-full-grid-on-mobile;
       @extend %span-full-grid-on-desktop;
-      @extend %display-block;
+      // @extend %display-block;
```

Comment:
Is this intended? Other columns do extend display block. Why is this one commented?

Reply:
not intended, thanks - fixed